readme.md

```diff
@@ -109,6 +109,14 @@ try {
 	}
 	*/
 }
+
+// Killing a spawned process gently, then with more force
+const subprocess = execa('node'); // Lets assume node doesn't react to SIGTERM
+setTimeout(() => {
+	subprocess.kill('SIGKILL', {
```

Comment:
```suggestion
subprocess.kill('SIGTERM', {
```

Reply:
Oh, this totally slipped. :s

Fixed: 
* 2fb0ae0a7664c4c240d3ae9f764b6b0976549827
* 233e222679fcbce4b1547370bffcd5bffe32c7c4